src/lib/viewers/doc/__tests__/DocBaseViewer-test.js

```diff
@@ -641,6 +644,68 @@ describe('src/lib/viewers/doc/DocBaseViewer', () => {
 
                 expect(startPreloadTimerStub).toBeCalled();
             });
+
+            test('should load preloader properly for doc first pages', () => {
+                jest.spyOn(docBase, 'getViewerOption').mockReturnValue(true);
```

Comment:
Do we need to reset these spies at all? Not sure if there is a global reset config enabled.

Reply:
Yea they are for the viewer option. I didn't want to continue with that sandbox pattern they were using as this was simpler IMO. You can't see it here but I added a restore mocks call to the afterEach function